docs/operator.md

```diff
@@ -242,6 +242,64 @@ Add the annotation to the AgentServiceConfig:
 oc annotate --overwrite AgentServiceConfig agent unsupported.agent-install.openshift.io/assisted-image-service-skip-verify-tls=true
 ```
 
+### Custom CA for Assisted Image Service OS Image Download
+
+It is possible to specify a CA to be used when downloading OS images. This should be used when OS images are being served from HTTPS servers where the CA would not typically be known to assisted installer.
+
+Start by creating a secret with a TLS certificate as the content, the key "tls.crt" should contain the certificate(s)
+The ConfigMap should be installed in the same namespace as the infrastructure-operator (ie. `multicluster-engine` or `assisted-installer` depending on how the infrastucture operator was deployed).
+
+```
+oc -n multicluster-engine create configmap image-service-additional-ca --from-file=/root/tls.crt
```

Comment:
Typically when looking at this repo we're not using the multicluster-engine namespace. I'd just leave the namespace parameter out here to avoid confusion.